lib/vquic/ngtcp2.c

```diff
@@ -889,12 +889,13 @@ static CURLcode qs_disconnect(struct quicsocket *qs)
 #elif defined(USE_GNUTLS)
```

Comment:
Should the `qs->qlogfd = -1` be added above while at it? What about `qs->sslctx = NULL` below and other fields?

Can you consider adding a comment here documenting the memory ownership and transfers?

Reply:
What do you mean with "memory ownership and transfers" ? If you can elaborate a little on what you're referring to here I'm sure I can clarify them with some comments. What memory has unclear ownership and what about the transfers?